src/app/shared/components/product/product-list/product-list.component.html

```diff
@@ -1,4 +1,17 @@
-<div class="product-list row">
+<div class="product-list row" style="position: relative;">
```

Comment:
no style statements in html, this is not much better class="position-static", but ok 
better to define it in the product-list.scss